flink-connectors/flink-connector-elasticsearch-base/src/main/java/org/apache/flink/streaming/connectors/elasticsearch/ElasticsearchUpsertTableSinkBase.java

```diff
@@ -319,6 +319,8 @@ private void validateKeyTypes(int[] keyFieldIndices) {
 	 * Keys for optional parameterization of the sink.
 	 */
 	public enum SinkOption {
+		CONNECTOR_USERNAME,
```

Comment:
We do not need `CONNECTOR_` prefix for `SinkOption`, I think just `USERNAME` and `PASSWORD` is fine.